client/plots/matrix.cells.js

```diff
@@ -169,24 +169,6 @@ export function setGeneVariantCellProps(cell, tw, anno, value, s, t, self, width
 			order: -1,
 			entry: { key: value.class, label: cell.label, fill: cell.fill, order, dt: value.dt, origin: value.origin }
 		}
-	} else if (value.dt >= dtgeneexpression) {
```

Comment:
regarding this improper use, i've added comments about dt usage in yesterday's branch but my commit was somehow erased.
i will add it to your branch

Reply:
committed

Reply:
thanks fixed now